### DISCUSSION TITLE
Error: "Failed to run sql query: must be owner of event trigger pgsodium_trg_mask_update"

### DISCUSSION THREAD

**Discussion by bennik88** [supabase/supabase#14716] [Questions] [answered]
### Promlem:
I'm trying to create a new column in my table but Supabase is giving me the following error message **"Failed to run sql query: must be owner of event trigger pgsodium_trg_mask_update"**.  *I also can't create tables or anything in general anymore.

### What I've tried:
I tried it in the Supabase table editor as well as in the SQL editor with 
```sql
alter table public.contacts add column notes text;
```
### Question:
I never had problems altering a table where does this issue suddenly come from and how can I solve it?

**Comment by GaryAustin1** [comment]
This  maybe from a similar issue on Discord: https://discord.com/channels/839993398554656828/1099367373112344656/1099504784257589358

```
drop view vault.decrypted_secrets;

drop extension supabase_vault;

drop extension pgsodium;

create extension pgsodium;

create extension supabase_vault;
```

**Reply by jiangbo2015** [reply]
Great, stuck here for a long time, finally solved

**Reply by asharejaz1** [reply]
I am getting the following error when running the query above :
Failed to run sql query: schema "vault" does not exist

Can you please tell me what should I do

**Comment by bennik88** [comment, marked as answer]
Thank you @GaryAustin1, I can confirm that after contacting the support they recommended me the same.

### First:
Run the below query, and if the result is 0 you should be able to run a series of commands I've included here.
```sql
select count(*) from vault.secrets -- if the result is 0, you should be able to run below
```

### If 0, Second:
```sql
drop view vault.decrypted_secrets;
drop extension supabase_vault;
drop extension pgsodium;
create extension pgsodium;
create extension supabase_vault;
```

I still don't know what could have caused this. The only additional information I got was the below.

> The pgsodium extension is enabled by default and was probably set up using the postgres role. There shouldn't be any security risk or anything breaking on your project.

Take care.

**Reply by asharejaz1** [reply]
I am getting the following error when running any one of the queries above :
Failed to run sql query: schema "vault" does not exist

Can you please tell me what should I do

**Reply by GaryAustin1** [reply]
Not really as I was just relaying other information.  But if vault schema does not exist, why do you thing this report is the same issue?
You may just want to run the pgsodium parts if it is that error.   But sort of risky randomly dropping things if you are not sure what is going on.
Are either of those extensions installed looking in the database extensions tab of the dashboard?
